builtin/mainmenu/tab_settings_simple.lua

```diff
@@ -234,11 +236,11 @@ local function handle_settings_buttons(this, fields, tabname, tabdata)
 		return true
 	end
 	if fields["cb_fancy_leaves"] then
-		core.settings:set("leaves_style", fields["cb_fancy_leaves"] and "fancy" or "opaque")
+		core.settings:set("leaves_style", (minetest.is_yes(fields["cb_fancy_leaves"]) and "fancy" or "opaque"))
 		return true
 	end
 	if fields["cb_touchtarget"] then
-		core.settings:set("touchtarget", fields["cb_touchtarget"])
+		core.settings:set("touchtarget", (minetest.is_yes(fields["cb_touchtarget"]) and "false" or "true"))
```

Comment:
```suggestion
		core.settings:set_bool("touchtarget", not minetest.is_yes(fields["cb_touchtarget"])))
```

Maybe it'd be better to rename `cb_touchtarget` to `cb_crosshair` as well?